### ISSUE TITLE
Notifications/errors do not disappear even after clearing them using Syncthing Web GUI

### ISSUE THREAD

**Issue by tomasz1986** [Martchus/syncthingtray#87]
**Relevant components**
* [x] Standalone tray application (based on Qt Widgets)
* [ ] Plasmoid/applet for Plasma desktop
* [ ] Dolphin integration
* [ ] Command line tool (`syncthingctl`)
* [ ] Integrated Syncthing instance (`libsyncthing`)
* [ ] Backend libraries

**Environment and versions**
SyncthingTray 1.1.2
* Qt version: all
* C++ compiler (name and version): N/A
* C++ standard library (name and version): N/A
* Operating system (name and version): Windows 10 (1809)

**Bug description**
When SyncthingTray reports a notification/error, the tray icon changes into the one with an exclamation mark. In this state, if we go to the Syncthing's Web GUI and resolve the issues there, they do not get cleared in SyncthingTray. Basically, to get rid of the error state, we need to open SyncthingTray specifically, click on "New notifications", and clear them there again.

**Steps to reproduce**
1. Start SyncthingTray and connect it to Syncthing.
2. Cause an error state on purpose, e.g. by deleting a folder in the OS.
3. Wait for SyncthingTray to change to the exclamation mark icon.
4. Open Syncthing's Web GUI and resolve the problem there.
5. SyncthingTray still remains in the error state.

**Expected behavior**
Now, I have no idea if there exists a simple solution for this, but it would be really nice if SyncthingTray could detect that Syncthing has already resolved the issue, and then clear its error state accordingly.

**Screenshots**

**Additional context**

**Comment by Martchus** [comment]
Is the folder itself still shown as broken? Or are you just talking about the overall status which still has the exclamation mark icon because there are unread notifications? If it is the latter the overall status should switch back as soon as you dismiss the notification in Syncthing Tray as well. I am not aware of a way to improve this. If the folder itself is really still shown as broken than that's a bug but I haven't noticed it so far.

**Comment by tomasz1986** [comment]
> Or are you just talking about the overall status which still has the exclamation mark icon because there are unread notifications? If it is the latter the overall status should switch back as soon as you dismiss the notification in Syncthing Tray as well. I am not aware of a way to improve this.

Yeah, this is the situation that I am talking about. Even though the notification has been dismissed in Syncthing itself, SyncthingTray requires to dismiss it separately again. It would be nice to be able to have it dismissed automatically, but is this even possible (i.e. to detect that the notification is no longer present in Syncthing)?

**Comment by Martchus** [comment]
When I remember correctly and nothing has changed in the last years it isn't possible. I also believe the same happens when one has multiple tabs of the official web UI open. There's https://docs.syncthing.net/rest/system-error-clear-post.html but I am not aware of any event (delivered via the event API) which would tell one that an error has been cleared.
